fobnail-attester.c

```diff
@@ -0,0 +1,158 @@
+/**
+ * Some LICENSE
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <arpa/inet.h>
+#include <coap3/coap.h>
+#include <signal.h>
+#include <stdbool.h>
+
+static bool quit = false;
+static const char LISTEN_ADDRESS[] = "0.0.0.0";
+static unsigned int port = COAP_DEFAULT_PORT; /* default port 5683 */
+
+static void handle_sigint(int signum)
+{
+	quit = true;
+}
+
+static void coap_attest_handler(struct coap_resource_t* resource, struct coap_session_t* session,
+				const struct coap_pdu_t* in, const struct coap_string_t* query,
+				struct coap_pdu_t* out)
+{
+	int ret;
+	char *res_buf = "Response from server.\n";
+	size_t res_buf_len = strlen(res_buf);
+
+	printf("Received message.\n");
+
+	/* prepare and send response */
+	ret = coap_add_data_large_response(resource,
+					   session,
+					   in,
+					   out,
+					   query,
+					   COAP_MEDIATYPE_APPLICATION_CBOR,
+					   -1,
+					   0,
+					   res_buf_len,
+					   res_buf,
+					   NULL,
+					   res_buf);
+	if (ret != 0)
```

Comment:
@0xDen Function returns 0 on failure and 1 on success.
```suggestion
	if (ret == 0)
```